ImageCapture/webcam_image_capture.py

```diff
@@ -0,0 +1,46 @@
+import cv2
+import os
+import time
+from datetime import datetime
+
+# Paramètres
+interval_seconds = 5  # Intervalle de temps entre chaque capture en secondes
+save_directory = "captured_images"  # Dossier où les images seront enregistrées
+
+# Création du dossier pour stocker les images, s'il n'existe pas
+if not os.path.exists(save_directory):
+    os.makedirs(save_directory)
+
+# Initialisation de la caméra (0 pour la caméra par défaut, essayez 1 si vous utilisez une autre webcam)
+cap = cv2.VideoCapture(0)
+
+if not cap.isOpened():
+    print("Erreur: Impossible d'accéder à la webcam.")
+    exit()

```

Comment:
You can add a parameter to the exit function. It is the status code. Usually 0 = OK, 1 = Error